lib/internal/freeze_intrinsics.js

```diff
@@ -20,21 +20,61 @@
 // https://github.com/tc39/proposal-ses/blob/e5271cc42a257a05dcae2fd94713ed2f46c08620/shim/src/freeze.js
 
 /* global WebAssembly, SharedArrayBuffer, console */
-/* eslint-disable no-restricted-globals */
 'use strict';
 
+const {
+  Array,
+  ArrayBuffer,
+  ArrayPrototypeForEach,
+  BigInt,
+  BigInt64Array,
+  BigUint64Array,
+  Boolean,
+  DataView,
+  Date,
+  Error,
+  EvalError,
+  Float32Array,
+  Float64Array,
+  Function,
+  Int16Array,
+  Int32Array,
+  Int8Array,
+  JSON,
```

Comment:
`JSON`, `Math` and `Reflect` don’t actually exist on `primordials`, only their methods.

---

I’m fixing that in <https://github.com/nodejs/node/pull/36016>.